app/views/applications_mailer/application_success.html.erb

```diff
@@ -312,31 +312,34 @@ Rails Girls has proven very popular, and we have more applicants than spots. Ple
-<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.thoughtworks.com/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">ThoughtWorks</a></span><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.canva.com" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Canva</a></span><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="http://hays.com.au/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Hays</a><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.travelwithjane.com" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Travel with Jane</a></span><br>
 <span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.blake.com.au" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Blake Education</a></span><br>
-<a href="https://travelwithjane.com/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Travel With Jane</a></span><br>
-<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.ruby.org.au" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Ruby Australia</a></span><br>
+<a href="http://www.lookahead.com.au/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">LookAhead Search</a><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.envato.com/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Envato</a></span><br>
+<a href="https://cultureamp.com/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Culture Amp</a></span><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.rea-group.com" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">REA</a></span><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.hired.com" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Hired</a></span><br>
 <br>
 We will be limiting the amount of email that you will get from us to the bare essentials, please make sure to watch your email for any updates.<br>
```

Comment:
These are just suggestions, but perhaps something along the lines of 
`We will be limiting the amount of emails being sent to you so please be sure to watch your inbox for any updates.`
`We will be limiting the amount of emails being sent to you so please be sure to keep an eye out`